bigbluebutton-html5/imports/utils/deviceInfo.js

```diff
@@ -10,6 +10,7 @@ const deviceInfo = {
 
     return {
       isPhone: smallSide <= MAX_PHONE_SHORT_SIDE,
+      isMobile: window.matchMedia('only screen and (max-width: 40em)').matches,
```

Comment:
This `isMobile` will fail if a phone like say Pixel XL is in Lanscape mode.

Reply:
This means Pixel XL is not a phone :robot: